data/language/en-GB.txt

```diff
@@ -3722,6 +3722,8 @@ STR_6258    :Clear (transparent)
 STR_6259    :Disabled
 STR_6260    :Show blocked tiles
 STR_6261    :Show wide paths
+STR_6262    :Always use system file browsing window
```

Comment:
I'm fine with the change, but then I'd like to see the existing usage of "system dialog window" adjusted too.

Reply:
Oh, I thought that the button in the load/save window was clear enough in context. Unless there are other usages I'm not aware of?

Reply:
"Always use system dialog window for saving/loading" fits the options window too, by the way.

Reply:
@AaronVanGeffen Possibly, but I like to use consistent terminology. I think it makes it easier for the user to understand they're referring to the same thing.

Reply:
"Always use system dialog window for file browsing", then?

Reply:
No, I meant changing it all to "system file browsing window". The existing button in the load/save window included.

Reply:
Done.

Reply:
Personally I think saying "System File Browser" would be better.  Browsing window has a really weird, sorta jilted cadence to it in my opinion.  Like calling a Chrome window a "Internet browsing window".

Reply:
I have no objections to that, but I'll let @AaronVanGeffen decide.

Reply:
Once more unto the breach, dear friends, once more…